project.clj

```diff
@@ -0,0 +1,26 @@
+(defproject amperity/vault-clj-aws "0.0.1-SNAPSHOT"
+  :description "Vault-clj extension to support aws iam instance authentication."
+  :url "https://github.com/amperity/vault-clj-aws"
+  :license {:name "Apache License"
+            :url "http://www.apache.org/licenses/LICENSE-2.0"}
+
+  :deploy-branches ["master"]
+  :pedantic? :abort
+
+  :dependencies
+  [[org.clojure/clojure "1.9.0"]
+   [org.clojure/tools.logging "0.4.0"]
+   [amperity/envoy "0.3.1"]
+   [amperity/vault-clj "0.6.6"]
+   [buddy/buddy-core "1.4.0"]
```

Comment:
Are you including this just for the Base64 code? I'd strongly recommend using Java's built in [`Base64`](https://docs.oracle.com/javase/8/docs/api/java/util/Base64.html) codec instead.

Reply:
nice, i'm not yet in the habit of remembering that lib.